vaadin-date-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/datetimepicker/demo/DateTimePickerView.java

```diff
@@ -36,79 +38,143 @@
 
     @Override
     public void initView() {
-        createSimpleDateTimePicker();
-        createLocaleChangeDateTimePicker();
-        addCard("Additional code used in the demo",
-                new Label("These methods are used in the demo."));
+        basicDemo(); // Basic usage
+        valueChangeEvent();
+        finnishDateTimePicker(); // Localizing
+        themeVariantsTextAlign(); // Theme variants
+        themeVariantsSmallSize();
+        styling(); // Styling
     }
 
-    private void createSimpleDateTimePicker() {
-        Div message = createMessageDiv("simple-picker-message");
-
+    private void basicDemo() {
+        Div div = new Div();
         // begin-source-example
-        // source-example-heading: Simple date time picker
+        // source-example-heading: Basic usage
         DateTimePicker dateTimePicker = new DateTimePicker();
 
-        dateTimePicker.addValueChangeListener(
-                event -> updateMessage(message, dateTimePicker));
+        DateTimePicker valueDateTimePicker = new DateTimePicker();
+        LocalDateTime now = LocalDateTime.now();
+        valueDateTimePicker.setValue(now);
```

Comment:
Nit: I guess we can leave only one `dateTimePicker` with pre-selected value or empty one in this demo 🤔 
Or we can split those in two as it's done on the client side.

Reply:
This was just based on the latest DatePicker demos which look like this 
![image](https://user-images.githubusercontent.com/75155/74226338-0dae0c80-4cc5-11ea-838a-ec4505d55b9c.png)

I'm fine with dropping either the empty one or the one with the value though. Not sure which would be better. But atm it's at least consistent with DatePicker demo which was recently updated.